### PR TITLE
Migrate schema tests

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Migrating and adapting the schema tests from legacy tests.

### CODE REVIEW

**Comment by jcrivenaes** [comment]
Where are the schema migrating from?

File: tests/test_schema/test_schema_logic.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+import yaml
+import json
+import jsonschema
+
+import pytest
+
+
+# pylint: disable=no-member
+
+logger = logging.getLogger(__name__)
```
Comment (jcrivenaes) [inline tests/test_schema/test_schema_logic.py:16]: Seems that logger are not in use

**Comment by perolavsvendsen** [comment]
> Where are the schema migrating from?

From legacy tests, updated the initial comment

**Review by jcrivenaes — APPROVED** [review]
LGTM

**Pull request merged**